tools/offcputime.py

```diff
@@ -61,6 +61,8 @@ def positive_nonzero_int(val):
     help="show stacks from user space only (no kernel space stacks)")
 stack_group.add_argument("-K", "--kernel-stacks-only", action="store_true",
     help="show stacks from kernel space only (no user space stacks)")
+parser.add_argument("-d", "--delimited", action="store_true",
```

Comment:
Per my comment in #559 and the plans regarding #545 we may want to be more verbose with the long version of this option. Possibly `--delimit-user-kernel-stacks` but I'm terrible at naming so there may be a better alternative haha. What do you think?